cloneable-modules/users/Controllers/Admin/UsersController.php

```diff
@@ -12,6 +13,59 @@ class UsersController extends AdminApiController
      */
     protected $controllerInfo = [
         'repository' => 'users',
-        'listOptions' => [],
+        'listOptions' => [
+            'select' => ['id','name','user_group_id','email'],
```

Comment:
Set it empty to allow the developer to determine what to select by him/herself.

Reply:
Do you think it will be good to fill it as default ?